Home_work7/Recursion.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+int factorial = 1;
+int factorialRecursion(int count) {
+	factorial = factorial * count;
+	if (count > 1) {
+		factorialRecursion(count - 1);
+	}
+	else {
+		std::cout << factorial << std::endl;
+		factorial = 1;
+	}
+	return 0;
+}
```

Comment:
in fact `factorial` is a global variable accessible everywhere. This is a bad approach
Better try using `return` value of the `factorialRecursion(...) call itself for muliplication with `count`